scss/tech.scss

```diff
@@ -42,7 +42,13 @@
     height: 100%;
     position: absolute;
     right: 5px;
+    transition: all 0,6s linear;
     width: 100%;
     z-index: -1;
   }
+
+  &:hover::before {
+    right: -5px;
```

Comment:
It isn't a good practice to animate this properties. Check this [article](https://medium.com/outsystems-experts/how-to-achieve-60-fps-animations-with-css3-db7b98610108)